source/activity/view-logs.txt

```diff
@@ -101,6 +112,7 @@ The following types are valid:
 - ``graphql``
 - ``sync``
 - ``schema``
+- ``errorhandler``
```

Comment:
Need to confirm this value once the [CLI type list](https://github.com/10gen/baas/blame/7911f2c40da976cbaa51f9d28f5f7bdd69309713/cli/commands/logs/list_inputs.go#L15-L15C35) is updated